content/master/concepts/providers.md

```diff
@@ -330,6 +330,166 @@ for a Provider.
 
 Each Provider determines their supported set of `args`.
 
+### Runtime configuration
+
+{{<hint "important" >}}
+`DeploymentRuntimeConfigs` is a beta feature. 
+
+It's on by default, and you could disable it by passing
+`--enable-deployment-runtime-configs=false` to the Crossplane deployment.
+{{< /hint >}}
+
+Runtime configuration is a generalized mechanism for configuring runtime for
+Crossplane packages with a runtime, namely `Providers` and `Functions`. It
+replaces the deprecated `ControllerConfig` type and is available in v1.14+.
```

Comment:
Perhaps "configuring the runtime". You could also just say "for configuring Crossplane packages with a runtime".